spec/addressable/idna_spec.rb

```diff
@@ -258,14 +258,18 @@
 end
 
 describe Addressable::IDNA, "when using the pure-Ruby implementation" do
-  before do
+  before :all do
```

Comment:
Small optimisation to avoid wasting useless CPU cycles reloading the whole files before EACH test. It's more than 10 times faster (but was already very fast so not a big deal)